app/helpers/format-date.js

```diff
@@ -1,4 +1,6 @@
 import Ember from "ember";
-export default Ember.Handlebars.makeBoundHelper(function(value) {
-	return moment(value).fromNow();
-});
\ No newline at end of file
+
+export default Ember.Helper.helper(function(params) {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).